crates/krun/src/bin/krun.rs

```diff
@@ -37,7 +38,16 @@ fn main() -> Result<()> {
         return Err(anyhow!("real user ID or effective user ID is 0"));
     }
 
-    let options = options().fallback_to_usage().run();
+    let options = options().run();
+
+    let lock_file = lock_or_connect(&options)?;
+    if lock_file.is_none() {
+        // If lock_or_connect has returned without error and without
+        // a lock file, it means there was already a krun instance
+        // running and we've requested it to launch the command
+        // successfully, so all the work is done.
```

Comment:
We should use an `enum` to represent the different cases instead of bespoke logic... :see_no_evil:

Reply:
Done.